### PR TITLE
refactored part that dependent on localization specific names

### PR DESCRIPTION
**Pull request by AmsTaFFix**
also added definitions to make easier write and read code

### CODE REVIEW

**Comment by AmsTaFFix** [comment]
@AlexFolland

File: AutoGear.lua [hunk 1]
```diff
@@ -2747,22 +2791,28 @@ function AutoGearReadItemInfo(inventoryID, lootRollID, container, slot, questRew
 					cannotUse = 1
 					reason = "(this spec should use a ranged weapon)"
 				end
-				if (weapons == "2hDW") then	--Alitwin: adding 2hdw
-					info.Slot = "MainHandSlot" info.SlotConst = INVSLOT_MAINHAND
-					info.Slot2 = "SecondaryHandSlot" info.Slot2Const = INVSLOT_OFFHAND
+				if (weapons == "2hDW") then
+					--Alitwin: adding 2hdw
```
Comment (AlexFolland) [inline AutoGear.lua:2795]: I think we can remove this comment now.

File: AutoGear.lua [hunk 2]
```diff
@@ -2853,22 +2909,17 @@ function AutoGearReadItemInfo(inventoryID, lootRollID, container, slot, questRew
...
 	if (info.Slot) then
 		info.shouldShowScoreInTooltip = 1
 	end
 	if (not cannotUse and (info.Slot or info.isMount)) then
 		info.Usable = 1
-	elseif classId == 15 then
+	elseif itemClassId == 15 then
 		cannotUse = 1
-		reason = "(classId = " .. classId .. ", it is not usable)"
+		reason = "(classId = " .. itemClassId .. ", it is not usable)"
```
Comment (AlexFolland) [inline AutoGear.lua:2919]: The `classId` part of this string doesn't match the new variable name.  Please update that to `itemClassId`.

**Review by AlexFolland — CHANGES_REQUESTED** [review]
Thanks for your pull request!  I'm requesting a few minor changes, and then I'll merge it.

**Pull request merged**